### PR TITLE
add notifications, use macOS like paths, new file to watch for WiFi changes, uninstall script

### PR DESCRIPTION
**Pull request by lisanet**
This pull request will merge some changes from other users too, mainly the addition of notifications.

The merge adds a more macOS like path for the config and script files and it moves all remaining configurable options to the config file.

There's also a new file to watch, so that it's possible to detect changing a WiFi network which the user doesn't want do be remembered.

A uninstall script and a few minor patches are included too (spelling, timestamp to scselect output, restrict size of log file)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,27 +1,96 @@
 # Location Changer
 
-It automatically changes OS X’s [network location](https://support.apple.com/en-us/HT202480)
-based on the name of Wi-Fi network and runs arbitrary scrips when it happens.
+*Please note, this is a fork of the original [Location Changer](https://github.com/eprev/locationchanger) by Anton Eprev*
```
Comment (aleksandr-vin) [inline README.md]: This is not for PR, right? ;)

Comment (aleksandr-vin) [reply README.md]: Same for installation url

Comment (lisanet) [reply README.md]: :-) oops, you're right.

**Comment by eprev** [comment]
Hey, @lisanet, sorry, I cannot accept your pull request due to the reasons mentioned previously in #10.

**Pull request closed without merging**